src/util/segmentation/operations/eraseInside.js

```diff
@@ -0,0 +1,59 @@
+import { getBoundingBoxAroundPolygon } from '../boundaries';
+import pointInPolygon from '../../pointInPolygon';
+
+import { getLogger } from '../../logger';
+
+const logger = getLogger('util:segmentation:operations:eraseInside');
+
+/**
+ * EraseInside - Erase all pixels labeled with the activeSegmentIndex,
+ * in the region defined by evt.operationData.points.
+ * @param  {} evt The Cornerstone event.
+ * @param {} evt.operationData An object containing the `pixelData` to
+ *                          modify, the `segmentIndex` and the `points` array.
+ * @returns {null}
+ */
+export default function eraseInside(evt) {
```

Comment:
While it's easy to just toss something the entire event, for very specific functions like this, it's better to pass exactly/only the data we need.

We don't do this in the BaseTool methods because you may want to override those, and we have no good way of knowing what data you'll want. We definitely don't want to constrain users either. But this particular use case allows us to more tightly control our input.

Reply:
Can't do that fam, this is a tool strategy.

Reply:
P.S. It'd make more sense for this to be named `eraseInsideFreehand`, I'll do that.

Reply:
Ah. It's more the file location that got me. `util`, to me, implies very generic general use. I guess I would expect these to be at `src/tools/internal` or `src/tools/segmentation/internal`

Reply:
Yeah it does feel a bit odd, I'm thinking `src/tools/segmentation/strategies` for anything that is actually a strategy, and `src/tools/segmentation/internal` for anything which is reused, but not a strategy?

Reply:
Refactored.